.github/ISSUE_TEMPLATE/01-bug-report.yml

```
@@ -0,0 +1,64 @@
+name: 🐞 Bug report
+description: Report a problem to help improve this project
+title: "[BUG] "
+labels: [bug, triage]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    attributes:
+      label: Problem description
+      description: |
+        A concise description of what you're experiencing.
+
+        Please explain:
+
+          * **what** you tried to achieve,
+          * **how** you went about it (referring to the code sample), and
+          * **why** the current behaviour is a problem and what output you expected instead.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Code sample
+      description: >
+        Create a [minimal, complete, verifiable example](https://stackoverflow.com/help/mcve).
+        Please, paste your code between the ``` tickmarks below or link to a gist.
```

```suggestion
        Please, paste your code between the ``` tickmarks below or link to a [gist](https://gist.github.com/).
```